Makefile

```diff
@@ -51,10 +52,38 @@ check-types: openfisca_core openfisca_web_api
 	@$(call help,$@:)
 	@mypy $?
 
-## Run openfisca-core tests.
+## Run openfisca-core & country/extension template tests.
 test: clean check-syntax-errors check-style check-types
+	@##	Usage:
+	@##
+	@##		make test [pytest_args="--ARG"] [openfisca_args="--ARG"]
+	@##
+	@##	Examples:
+	@##
+	@##		make test
+	@##		make test pytest_args="--exitfirst"
+	@##		make test openfisca_args="--performance"
+	@##		make test pytest_args="--exitfirst" openfisca_args="--performance"
+	@##
+	@$(call help,$@:)
+	@${MAKE} test-core
+	@${MAKE} test-country
+	@${MAKE} test-extension
```

Comment:
Why is this not simply added as a dependent target to `test`, like `clean` etc? 🤔